isaacgymenvs/tasks/franka_mp_full.py

```diff
@@ -596,13 +622,22 @@ def launch_test(cfg: DictConfig):
     total_error = 0
     num_failed_plans = 0
     num_plans = 1000
+    def check_input():
+        # Check if there's any input available
+        if select.select([sys.stdin], [], [], 0)[0]:
+            # Read the input (including the newline character)
+            line = sys.stdin.readline()
+            return True
+        return False
```

Comment:
this is just to keep the simulation rendering and enable us to move around using the mouse.
To reset we press enter